src/app/webserver/files/webfile-list.ts

```diff
@@ -308,7 +305,7 @@ export class WebFileListComponent implements OnInit, OnDestroy {
     }
 
     private filter() {
-        this._svc.files.take(1).subscribe(files => {
+        this._svc.files.pipe(take(1)).subscribe(files => {
```

Comment:
This looks like a legacy code. For safety, please check if we need to call unsubscribe or not.